### PR TITLE
util: add release version info

### PR DESCRIPTION
**Pull request by overvenus**
Add version information in tikv's log and tikv-server -V output.

CC pingcap/tidb#4218

### CODE REVIEW

**Comment by siddontang** [comment]
LGTM

File: src/bin/tikv-server.rs [hunk 1]
```diff
@@ -274,8 +274,11 @@ fn main() {
     let long_version: String = {
         let (hash, branch, time, rust_ver) = util::build_info();
         format!(
-            "{}\nGit Commit Hash:   {}\nGit Commit Branch: {}\nUTC Build Time:    {}\nRust \
-             Version:      {}",
+            "\nRelease Version:   {}\
```
Comment (BusyJay) [inline src/bin/tikv-server.rs]: Is the leading `\n` necessary?

Comment (overvenus) [inline src/bin/tikv-server.rs]: For constructing a message like:
```
TiKV 
Release Version:   0.0.1
Git Commit Hash:   31b8cee0513ba45ab1126cf1a210a0003464e493
Git Commit Branch: ov/release-info
UTC Build Time:    2017-08-22 12:03:34
Rust Version:      1.21.0-nightly (215e0b10e 2017-08-08)
```

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**